### DISCUSSION TITLE
请教一个关于Facebook Oauth的问题

### DISCUSSION THREAD

**Discussion by medmin** [walinejs/waline#1542] [Q&A] [answered]
请教一个问题，这个facebook的登录，需要一个所谓的privacy policy和terms of service
这个怎么弄？

我看lithub.cc这个网站，但打不开，本来想着是看看这个网站怎么弄的

谢谢

**Comment by lizheming** [comment, marked as answer]
我这里直接就填的 OAUTH_URL。另外官方的 Facebook 登录已经可以了。
